gdx/src/com/badlogic/gdx/scenes/scene2d/actions/Actions.java

```diff
@@ -22,17 +22,54 @@
 import com.badlogic.gdx.scenes.scene2d.Actor;
 import com.badlogic.gdx.scenes.scene2d.EventListener;
 import com.badlogic.gdx.scenes.scene2d.Touchable;
+import com.badlogic.gdx.utils.DefaultPool.PoolSupplier;
 import com.badlogic.gdx.utils.Null;
 import com.badlogic.gdx.utils.Pool;
-import com.badlogic.gdx.utils.Pools;
+import com.badlogic.gdx.utils.PoolManager;
 
 /** Static convenience methods for using pooled actions, intended for static import.
  * @author Nathan Sweet */
 public class Actions {
 
+	public static final PoolManager ACTION_POOL_MANAGER = new PoolManager();
```

Comment:
This one is public and sometimes use with `import static` so `ACTION_` makes sense, but can it be `ACTION_POOLS`?

Generic suffixes like "controller" or "manager" don't add much IMHO. `PoolManager` makes sense for the class name and differentiates it from other classes, but fields of type `PoolManager` don't necessarily need "manager" in their names.